internal/handlers/pg/pgdb/collections.go

```diff
@@ -196,10 +190,10 @@ func CreateCollectionIfNotExist(ctx context.Context, querier pgxtype.Querier, db
 
 // DropCollection drops FerretDB collection.
 //
-// It returns (possibly wrapped) ErrTableNotExist if schema or table does not exist.
+// It returns (possibly wrapped) ErrTableNotExist if database or collection does not exist.
```

Comment:
There is some confusion here. The function is called `DropCollection` and the comment contains `database/colletion` things, but the error is called `ErrTableNotExist`... It's probably ok, but not 100% consistent.

Reply:
Yeah, we will fix it later. One step at a time

Reply:
Makes sense!